src/testUtils/shallow.jsx

```diff
@@ -28,9 +28,11 @@ class ReactShallowRenderer {
     try {
       out = new ReactShallowRenderer(Component);
     } catch (error) {
-      console.log({ error, Component });
+      if (!error.message.includes('ReactShallowRenderer')) {
```

Comment:
What is the reason for this?

Reply:
So, Shallow Rendering doesn't work on basic jsx (`<div><h1>Hi</h1></div>`), and when we try and it fails for that reason, we provide a flat render.  *BUT* we don't want to swallow other errors.  This just swallows the "shallow rendering doesn't work on jsx" error pretty much, while allowing errors in the code/test to raise errors appropriately.